packages/bsky/Dockerfile

```diff
@@ -18,18 +18,20 @@ COPY ./packages/repo ./packages/repo
 COPY ./packages/uri ./packages/uri
 COPY ./packages/xrpc ./packages/xrpc
 COPY ./packages/xrpc-server ./packages/xrpc-server
-RUN ATP_BUILD_SHALLOW=true yarn install --frozen-lockfile > /dev/null
-RUN yarn workspaces run build --update-main-to-dist > /dev/null
+
+RUN npm install -g pnpm
```

Comment:
It may make sense to hoist this up towards the top of the file so that it's nearly always in the docker cache.